### PR TITLE
AsideSection component

### PR DESCRIPTION
**Pull request by swese44**
Creates the AsideSection component, for use in our sidebar

## Pull request checklist

* [x] Component `README.md` file is up-to-date.
* [x] Component is unit tested.

### CODE REVIEW

**Comment by jessemartin** [comment]
👍

**Comment by swese44** [comment]
hey @rahultiru i added a demo example with a list of MediaObjects

File: src/components/AsideSection/AsideSection.test.tsx [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  describe('with action prop', () => {
+    beforeEach(() => {
+      const action = <Clickable>Add</Clickable>;
+      component = shallow(<AsideSection title="Test title" action={action}/>);
+    });
+
+    it('renders the action in its own FixedGridColumn', () => {
+      expect(component).toMatchSnapshot();
+    });
+
```
Comment (richardtagger) [inline src/components/AsideSection/AsideSection.test.tsx]: too many line breaks here

File: src/components/AsideSection/AsideSection.test.tsx [hunk 2]
```diff
@@ -0,0 +1,46 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import * as React from 'react';
+import { shallow, ShallowWrapper } from 'enzyme';
+import Clickable from '../Clickable';
+import AsideSection, { AsideSectionProps } from './index';
```
Comment (richardtagger) [inline src/components/AsideSection/AsideSection.test.tsx]: from '.'

**Comment by swese44** [comment]
👍  i'm following up to adjust the bottomSpacing instead of px margin

**Comment by swese44** [comment]
@unindented travis seems to fail for no reason in less than 5 minutes each run...

**Comment by swese44** [comment]
looks like chrome errors were logged this time around

**Comment by unindented** [comment]
What's the f**** point of Docker if builds are not repeatable? Why is this failing now?

**Comment by unindented** [comment]
Updated the Dockerfile to explicitly install all dependencies mentioned in https://github.com/GoogleChrome/puppeteer/blob/master/docs/troubleshooting.md#chrome-headless-doesnt-launch

Hope it lasts.

**Pull request merged**